src/redisenterprise/azext_redisenterprise/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
-    "azext.minCliCoreVersion": "2.61.0",
-    "azext.isPreview": true
-}
+    "azext.isPreview": true,
+    "azext.minCliCoreVersion": "2.70.0"
+}
```

Comment:
@AllyW Can you help confirm if this is allowed for a preview extension upgrading from `1.2.1b2` to `1.3.0`?

Reply:
module owner can choose to release a new preview or stable version as it suits their business justification. However the next stable of `1.2.1b2` is `1.2.1` and `azext.isPreview: true` should be removed in  `azext_metadata.json`